strawberryfields/gbs/similarity.py

```diff
@@ -156,7 +165,7 @@ def orbit_to_sample(orbit: list, modes: int) -> list:
     if modes < len(orbit):
         raise ValueError("Number of modes cannot be smaller than length of orbit")
 
-    sample = orbit + [0] * (modes - len(orbit))
+    sample = list(orbit) + [0] * (modes - len(orbit))

```

Comment:
Checking that the input is a valid orbit may be a good idea. However, I believe that adding zeros here won't result in an issue since the added zeros `[0]* (modes - len(orbit))` takes the length of the orbit into account.

If the input `orbit` is a list, why do we need `list(orbit)`?

Reply:
```suggestion
    sample = orbit + [0] * (modes - len(orbit))
```
@ixfoduap helped answer my questions in person 😁
And agreed, shall we remove the `list` part then?